Description: background.vala: Fix build with vala 0.41.91 (no default
 value allowed when using custom getter.)
Author: Michael Webster <miketwebster@gmail.com>
Commit: ed52e2b1795e262dd52a1e1ccc2490a9a20aa70a
Last Update: 2018-08-16

---
 src/background.vala | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/src/background.vala b/src/background.vala
index 0fe0bee..f0789bf 100644
--- a/src/background.vala
+++ b/src/background.vala
@@ -443,8 +443,14 @@ public class Background : Gtk.Fixed

     private string _current_background;
     public string? current_background {
-        get { return _current_background; }
+        get {
+            if (_current_background == null)
+            {
+                _current_background = fallback_color;
+            }

+            return _current_background;
+            }
         set {
             if (value == null || value == "")
             {
@@ -456,8 +462,6 @@ public class Background : Gtk.Fixed

             reload ();
         }
-
-        default = fallback_color;
     }

     public bool draw_grid { get; set; default = true; }
--
2.17.1

